pkg/agent/utils.go

```diff
@@ -194,6 +195,40 @@ func getBase64EncodedGzippedCustomScript(csFilename string, config *datamodel.No
 	return getBase64EncodedGzippedCustomScriptFromStr(csStr)
 }
 
+func removeComments(b []byte) []byte {
```

Comment:
I know it's a pain, but we should probably write some basic unit tests for this

Reply:
true, will add some